game/audio.cc

```diff
@@ -407,12 +413,12 @@ struct Output {
 };
 
 
-Device::Device(unsigned int in, unsigned int out, double rate, unsigned int dev):
+Device::Device(int in, int out, double rate, PaDeviceIndex dev):
```

Comment:
I know pipewire supports pulseaudio protocol, but having specific mentions of pulseaudio in the code does kind of get on my nerves...

Reply:
What?

Reply:
I'm currently reworking the libportaudio part, and this modification is just about making clear the dev was the PA index not a random int (dependency on PA was hidden behind a bad cast). So this may look like an issue, but that's just because it reveals that the former code was not good anyway.